sysl2/sysl/commands/protobuf.go

```diff
@@ -0,0 +1,50 @@
+package commands
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/anz-bank/sysl/sysl2/sysl/pbutil"
+	"github.com/sirupsen/logrus"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+type protobuf struct {
+	output string
+	mode   string
+}
+
+func (p *protobuf) Name() string            { return "protobuf" }
+func (p *protobuf) RequireSyslModule() bool { return true }
+
+func (p *protobuf) Init(app *kingpin.Application) *kingpin.CmdClause {
```

Comment:
I prefer a method name "Configure...." rather than Init(), maintains the readability of the code.

Reply:
done